src/library/Uno.Material/Styles/Controls/v1/Button.xaml

```diff
@@ -4,11 +4,12 @@
 					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:um="using:Uno.Material"
+					xmlns:ut="using:Uno.Themes"
 					xmlns:not_win="http://uno.ui/not_win"
 					mc:Ignorable="d not_win">
 
 	<!-- Documented variables -->
-	<um:ToUpperConverter x:Key="ToUpperConverter" />
+	<!--<ut:ToUpperConverter x:Key="ToUpperConverter" />-->
```

Comment:
This one stays commented @pictos ? why ?

Reply:
Good catch, that was a test that I did, because that converter is already defined on `Converters.xaml` file, so there's no need to instantiate it again, just reference with `StaticResource`, it built but is that correct? Or should I bring it back?